src/main/java/com/sokol/webgame/webgame/service/GameService.java

```diff
@@ -37,4 +48,22 @@ public List<LevelsDto> getLevels() {
                 .map(levelsMapping::map)
                 .toList();
     }
+
+    public EnemyBackgroundDto getLevel(Enemy type) {
+        Background randomBackground = backgroundRepo.findRandomBack(); // Рандомний фон
+        Enemy randomEnemy;
+
+        if (type.getType().equalsIgnoreCase("Enemy")) {
+            randomEnemy = enemyRepo.findRandomEnemy(); // Рандомний ворог
+        } else {
+            randomEnemy = enemyRepo.findRandomBoss();  // Рандомний бос
+        }
+
+        EnemyBackgroundDto result = new EnemyBackgroundDto();
```

Comment:
Take a look at @Builder annotation from Lombok. I would like to see the following:
```suggestion
        return EnemyBackgroundDto.builder()
            .randomEntity(getRandomEntity(type))
            .background(backgroundRepo.findRandomBack())
            .build();
```